### DISCUSSION TITLE
v1.0.9

### DISCUSSION THREAD

**Discussion by fdivitto** [fdivitto/FabGL#272] [Announcements]
- Added PC8250 class (Universal Asynchronous ReceiverITransmitter emulator)
- Terminal: removed support for Arduino and internal serial port
- Terminal: added support for BREAK signal
- added BASIC demo programs as ASCII files
- fixed modeline parser (doesn't parse additional parameters when V and H sync is not specified)
- Display controllers: added support for double buffering suspension
- Added support for TTGO T-Display V1.1 (class TTGOTDisplayV11Controller)
- img2bitmap, improved color extension and python3 friendly
- Added SerialPort class
- Added SerialPortTerminalConnector class
- Multitasking CP/M: support for out-of-terminal serial port
- Multitasking CP/M: added TERM USB new command
- AnsiTerminal: added new baud rates
- AnsiTerminal: fixed crash if no mouse is configured
- AnsiTerminal: use of PS/2 port as serial port (5V levels, RX/TX only)
- PC Emulator: allow to download of images from https and from redirects
- PC Emulator: added experimental support for serial port
- PC Emulator: fixed BIOS int14 initialization bug
- PC Emulator: increased SD card speed on running machine (without WiFi)
- PC Emulator: improved disk change support
- PC Emulator: added "Restart Machine" option in SysReq menu
- removed unnecessary IRAM code (allow to compile with core 2.0.4)
- improved core 2.0.0 compatibility

<hr /><em>This discussion was created from the release <a href='https://github.com/fdivitto/FabGL/releases/tag/v1.0.9'>v1.0.9</a>.</em>

**Comment by alexuren78** [comment]
Добрый день, а сделайте bin файл этого релиза
